src/library-authoring/add-content/AddContent.tsx

```diff
@@ -37,7 +37,7 @@ import { ContainerType } from '../../generic/key-utils';
 
 type ContentType = {
   name: string,
-  disabled: boolean,
+  disabled?: boolean,
   icon?: React.ComponentType,
```

Comment:
```suggestion
```
Can we delete this `icon` field now? Seems to be unused after these changes.

Reply:
Ah, I made a mistake here -- we do need that `icon` parameter so we can show the little sparkle next to "Advanced / Other": 

![image](https://github.com/user-attachments/assets/af2ac9e0-11c8-408a-8a9b-b1c646807842)

Fixed with d4ffcf97e3c116b698ded6e110fb9258bd990bf5.